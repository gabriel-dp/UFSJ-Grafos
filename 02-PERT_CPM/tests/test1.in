1 2 7
1 3 5
2 4 9
3 5 6
4 5 0
4 6 11
5 7 4
6 8 3
7 8 8
8 9 6
8 10 4
10 9 0
9 11 7
