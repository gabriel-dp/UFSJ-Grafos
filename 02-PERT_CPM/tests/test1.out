| ## | Min | Max | Slack |
|  1 |   0 |   0 |     0 |
|  2 |   7 |   7 |     0 |
|  3 |   5 |  12 |     7 |
|  4 |  16 |  16 |     0 |
|  5 |  16 |  18 |     2 |
|  6 |  27 |  27 |     0 |
|  7 |  20 |  22 |     2 |
|  8 |  30 |  30 |     0 |
|  9 |  36 |  36 |     0 |
| 10 |  34 |  36 |     2 |
| 11 |  43 |  43 |     0 |

Critical path:
( 1) -> ( 2) -> ( 4) -> ( 6) -> ( 8) -> ( 9) -> (11)